src/conf_mode/interfaces-wwan.py

```diff
@@ -101,6 +101,9 @@ def verify(wwan):
     verify_vrf(wwan)
     verify_mirror_redirect(wwan)
 
+    if ('ping-check' in wwan) and ('enable' in wwan['ping-check']) and ('address' not in wwan['ping-check']):
```

Comment:
Brackets aren't really necessary here

Reply:
`ping-check` will not be available as dict key in wwan as the hyphen is mangled to _.

Please use:

```python
if 'ping_check' in wwan and 'address' not in wwan['ping_check']:
    raise ConfigError(f'Ping check for "{ifname}" requires address to be defined!')
```

Reply:
> `ping-check` will not be available as dict key in wwan as the hyphen is mangled to _.
> 
> Please use:
> 
> ```python
> if 'ping_check' in wwan and 'address' not in wwan['ping_check']:
>     raise ConfigError(f'Ping check for "{ifname}" requires address to be defined!')
> ```

Has this always been the case?  
Does it means that my testing so far were not really happen?  

If that is the case, I've to re-test everything first.

Reply:
Yes

```python
def get_config(config=None):
    """
    Retrive CLI config as dictionary. Dictionary can never be empty, as at least the
    interface name will be added or a deleted flag
    """
    if config:
        conf = config
    else:
        conf = Config()
    base = ['interfaces', 'wwan']
    ifname, wwan = get_interface_dict(conf, base)
```

As the `waan` dict is retrieved using our general `get_interface_dict()` call - all hyphens are replaced by underscore. See https://github.com/vyos/vyos-1x/blob/current/python/vyos/configdict.py#L414C58-L414C58 for the implementation